Userland/Games/BrickGame/BrickGame.cpp

```diff
@@ -645,10 +647,10 @@ void BrickGame::paint_game(GUI::Painter& painter, Gfx::IntRect const& rect)
                 paint_cell(painter, cell_rect(position), (*m_brick_game)[board_position]);
             }
 
-        paint_sidebar_text(painter, 0, String::formatted("Score: {}", m_brick_game->score()).release_value_but_fixme_should_propagate_errors());
-        paint_sidebar_text(painter, 1, String::formatted("Level: {}", m_brick_game->level()).release_value_but_fixme_should_propagate_errors());
-        paint_sidebar_text(painter, 4, String::formatted("Hi-Score: {}", m_high_score).release_value_but_fixme_should_propagate_errors());
-        paint_sidebar_text(painter, 12, "Next:"sv);
+        (void)paint_sidebar_text(painter, 0, TRY(String::formatted("Score: {}", m_brick_game->score())));
+        (void)paint_sidebar_text(painter, 1, TRY(String::formatted("Level: {}", m_brick_game->level())));
+        (void)paint_sidebar_text(painter, 4, TRY(String::formatted("Hi-Score: {}", m_high_score)));
+        (void)paint_sidebar_text(painter, 12, "Next:"sv);
```

Comment:
Sidenote, if you _were_ to have `ErrorOr<void> ::paint_sidebar_text()`, you'd then usually `TRY(...)` all these statements instead of `(void)`ing them out.